.github/workflows/cli-test-prepare-packages.yaml

```diff
@@ -0,0 +1,110 @@
+name: Prepare and call the CLI testing - stub
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      zksolc:
+        type: string
+        description: "zkSolc version, (repo: https://github.com/matter-labs/zksolc-bin/raw/main) e.g. v1.3.22"
+        required: true
+        default: "v1.3.22"
+      solc:
+        type: string
+        description: "solc version, (repo: https://github.com/ethereum/solc-bin/raw/gh-pages) e.g. v0.8.22+commit.4fc1097e "
```

Comment:
Please switch to the release repo of our solc fork: https://github.com/matter-labs/era-solidity/releases

Reply:
done